docs/pages/development/installation.md

```diff
@@ -8,36 +8,36 @@ import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
 import { Tab, Tabs } from '~/components/plugins/Tabs';
 import { Terminal } from '~/ui/components/Snippet';
 
-The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or Bare project. To add a the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
+> Note: These docs assume a basic level of familiarity with React-Native. If you are new to React-Native, please see the [React-Native docs](https://reactnative.dev/docs/getting-started).
```

Comment:
```suggestion
> Note: These docs assume a basic level of familiarity with React Native. If you are new to React Native, learn how to get started with [React Native](https://reactnative.dev/docs/getting-started).
```

For consistency reasons, let's use "React Native" without the hyphen. Also, React Native's branding doesn't use hyphen.